lib/llm/Cargo.toml

```diff
@@ -80,7 +80,7 @@ xxhash-rust = { workspace = true }
 akin = "0.4.0"
 blake3 = "1"
 bytemuck = "1.22"
-candle-core = { version = "0.8.0" }
+candle-core = { version = "0.9.1" }
```

Comment:
> I tried to do the single-version-of-candle, but realised it's not possible because [mistral.rs](http://mistral.rs/) forks it

Would it make sense to install same fork here to reduce the duplication, and inline a comment about it?

https://github.com/EricLBuehler/mistral.rs/blob/53ebc7480cd08b124bc99ca9f6365af6152c6fa7/Cargo.toml#L30

Reply:
Humm, possibly. We only use candle for GGUF handle, and that code is adapted from mistral.rs originally.